+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
Python 3.11.6
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Sat Feb 15 13:55:37 2025 - INFO  - Start to prepare the test environment.
+ grep -v SP1 /etc/openEuler-release
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Sat Feb 15 13:55:37 2025 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Sat Feb 15 13:55:38 2025 - INFO  - Start to run test.
+ hostnamectl
+ grep Virtualization
+ test_execution cpupower.service
+ service=cpupower.service
++ date '+%Y-%m-%d %T'
+ log_time='2025-02-15 13:55:39'
+ test_restart cpupower.service
+ service=cpupower.service
+ systemctl restart cpupower.service
Job for cpupower.service failed because the control process exited with error code.
See "systemctl status cpupower.service" and "journalctl -xeu cpupower.service" for details.
+ CHECK_RESULT 1 0 0 'cpupower.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='cpupower.service restart failed'
+ exit_mode=0
+ '[' -z '1 0 0 cpupower.service restart failed' ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'cpupower.service restart failed'
+ LOG_ERROR 'cpupower.service restart failed'
+ message='cpupower.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'cpupower.service restart failed'
Sat Feb 15 13:55:40 2025 - ERROR - cpupower.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 34'
+ message='../common/common_lib.sh line 34'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 34'
Sat Feb 15 13:55:40 2025 - ERROR - ../common/common_lib.sh line 34
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status cpupower.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'cpupower.service restart failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='cpupower.service restart failed'
+ exit_mode=0
+ '[' -z '1 0 0 cpupower.service restart failed' ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'cpupower.service restart failed'
+ LOG_ERROR 'cpupower.service restart failed'
+ message='cpupower.service restart failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'cpupower.service restart failed'
Sat Feb 15 13:55:46 2025 - ERROR - cpupower.service restart failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 37'
+ message='../common/common_lib.sh line 37'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 37'
Sat Feb 15 13:55:46 2025 - ERROR - ../common/common_lib.sh line 37
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl stop cpupower.service
+ CHECK_RESULT 0 0 0 'cpupower.service stop failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='cpupower.service stop failed'
+ exit_mode=0
+ '[' -z '0 0 0 cpupower.service stop failed' ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status cpupower.service
+ grep 'Active: inactive'
+ CHECK_RESULT 1 0 0 'cpupower.service stop failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='cpupower.service stop failed'
+ exit_mode=0
+ '[' -z '1 0 0 cpupower.service stop failed' ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'cpupower.service stop failed'
+ LOG_ERROR 'cpupower.service stop failed'
+ message='cpupower.service stop failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'cpupower.service stop failed'
Sat Feb 15 13:55:52 2025 - ERROR - cpupower.service stop failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 42'
+ message='../common/common_lib.sh line 42'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 42'
Sat Feb 15 13:55:52 2025 - ERROR - ../common/common_lib.sh line 42
+ '[' 0 -eq 1 ']'
+ return 0
+ systemctl start cpupower.service
Job for cpupower.service failed because the control process exited with error code.
See "systemctl status cpupower.service" and "journalctl -xeu cpupower.service" for details.
+ CHECK_RESULT 1 0 0 'cpupower.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='cpupower.service start failed'
+ exit_mode=0
+ '[' -z '1 0 0 cpupower.service start failed' ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'cpupower.service start failed'
+ LOG_ERROR 'cpupower.service start failed'
+ message='cpupower.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'cpupower.service start failed'
Sat Feb 15 13:55:53 2025 - ERROR - cpupower.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 44'
+ message='../common/common_lib.sh line 44'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 44'
Sat Feb 15 13:55:53 2025 - ERROR - ../common/common_lib.sh line 44
+ '[' 0 -eq 1 ']'
+ return 0
+ SLEEP_WAIT 5
+ wait_time=5
+ cmd=
+ mode=1
+ python3 /root/mugen/libs/locallibs/sleep_wait.py --time 5 --cmd '' --mode 1
+ systemctl status cpupower.service
+ grep 'Active: active'
+ CHECK_RESULT 1 0 0 'cpupower.service start failed'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='cpupower.service start failed'
+ exit_mode=0
+ '[' -z '1 0 0 cpupower.service start failed' ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'cpupower.service start failed'
+ LOG_ERROR 'cpupower.service start failed'
+ message='cpupower.service start failed'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'cpupower.service start failed'
Sat Feb 15 13:55:59 2025 - ERROR - cpupower.service start failed
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 47'
+ message='../common/common_lib.sh line 47'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 47'
Sat Feb 15 13:56:00 2025 - ERROR - ../common/common_lib.sh line 47
+ '[' 0 -eq 1 ']'
+ return 0
+ test_enabled cpupower.service
+ service=cpupower.service
++ systemctl is-enabled cpupower.service
+ state=disabled
+ '[' disabled == enabled ']'
+ '[' disabled == disabled ']'
++ systemctl enable cpupower.service
++ grep 'Created symlink'
++ awk '{print $3}'
++ tr -d '"'
+ symlink_file=/etc/systemd/system/multi-user.target.wants/cpupower.service
+ find /etc/systemd/system/multi-user.target.wants/cpupower.service
/etc/systemd/system/multi-user.target.wants/cpupower.service
+ CHECK_RESULT 0 0 0 'cpupower.service enable failed'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='cpupower.service enable failed'
+ exit_mode=0
+ '[' -z '0 0 0 cpupower.service enable failed' ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl disable cpupower.service
Removed "/etc/systemd/system/multi-user.target.wants/cpupower.service".
+ find /etc/systemd/system/multi-user.target.wants/cpupower.service
find: ‘/etc/systemd/system/multi-user.target.wants/cpupower.service’: No such file or directory
+ CHECK_RESULT 1 0 1 'cpupower.service disable failed'
+ actual_result=1
+ expect_result=0
+ mode=1
+ error_log='cpupower.service disable failed'
+ exit_mode=0
+ '[' -z '1 0 1 cpupower.service disable failed' ']'
+ '[' 1 -eq 0 ']'
+ test 1x == 0x
+ return 0
+ journalctl --since '2025-02-15 13:55:39' -u cpupower.service
+ grep -i 'fail\|error'
+ grep -v -i 'DEBUG\|INFO\|WARNING'
Feb 15 13:55:39 localhost.localdomain cpupower[7882]: Error setting new values. Common errors:
Feb 15 13:55:39 localhost.localdomain systemd[1]: cpupower.service: Failed with result 'exit-code'.
Feb 15 13:55:39 localhost.localdomain systemd[1]: Failed to start Configure CPU power related settings.
Feb 15 13:55:53 localhost.localdomain cpupower[7901]: Error setting new values. Common errors:
Feb 15 13:55:53 localhost.localdomain systemd[1]: cpupower.service: Failed with result 'exit-code'.
Feb 15 13:55:53 localhost.localdomain systemd[1]: Failed to start Configure CPU power related settings.
+ CHECK_RESULT 0 0 1 'There is an error message for the log of cpupower.service'
+ actual_result=0
+ expect_result=0
+ mode=1
+ error_log='There is an error message for the log of cpupower.service'
+ exit_mode=0
+ '[' -z '0 0 1 There is an error message for the log of cpupower.service' ']'
+ '[' 1 -eq 0 ']'
+ test 0x == 0x
+ test -n 'There is an error message for the log of cpupower.service'
+ LOG_ERROR 'There is an error message for the log of cpupower.service'
+ message='There is an error message for the log of cpupower.service'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'There is an error message for the log of cpupower.service'
Sat Feb 15 13:56:04 2025 - ERROR - There is an error message for the log of cpupower.service
+ (( exec_result++ ))
+ LOG_ERROR '../common/common_lib.sh line 28'
+ message='../common/common_lib.sh line 28'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message '../common/common_lib.sh line 28'
Sat Feb 15 13:56:05 2025 - ERROR - ../common/common_lib.sh line 28
+ '[' 0 -eq 1 ']'
+ return 0
+ test_reload cpupower.service
+ service=cpupower.service
+ systemctl start cpupower.service
Job for cpupower.service failed because the control process exited with error code.
See "systemctl status cpupower.service" and "journalctl -xeu cpupower.service" for details.
+ systemctl cat cpupower.service
+ grep -q ExecReload
+ systemctl reload cpupower.service
+ grep 'Job type reload is not applicable'
Failed to reload cpupower.service: Job type reload is not applicable for unit cpupower.service.
+ CHECK_RESULT 0 0 0 'Job type reload is not applicable for unit cpupower.service'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Job type reload is not applicable for unit cpupower.service'
+ exit_mode=0
+ '[' -z '0 0 0 Job type reload is not applicable for unit cpupower.service' ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ systemctl status cpupower.service
+ grep 'Active: active'
+ systemctl status cpupower.service
+ grep 'inactive (dead)'
+ return 1
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Sat Feb 15 13:56:05 2025 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 6
+ test 6 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Sat Feb 15 13:56:06 2025 - ERROR - failed to execute the case.
+ exit 6
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Sat Feb 15 13:56:06 2025 - INFO  - Start to restore the test environment.
+ grep -v SP1 /etc/openEuler-release
+ LOG_INFO 'Finish to restore the test environment.'
+ message='Finish to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Finish to restore the test environment.'
Sat Feb 15 13:56:07 2025 - INFO  - Finish to restore the test environment.
